malariagen_data/anoph/base.py

```diff
@@ -33,56 +47,97 @@ class base_params:
         names.
         """,
     ]
-    region: TypeAlias = Annotated[
-        Union[str, Region],
+
+    single_region: TypeAlias = Annotated[
+        single_region_param_type,
         """
         Region of the reference genome. Can be a contig name, region string
         (formatted like "{contig}:{start}-{end}"), or identifier of a genome
         feature such as a gene or transcript.
         """,
     ]
+
+    region: TypeAlias = Annotated[
+        region_param_type,
+        """
+        Region of the reference genome. Can be a contig name, region string
+        (formatted like "{contig}:{start}-{end}"), or identifier of a genome
+        feature such as a gene or transcript. Can also be a sequence (e.g., list)
+        of regions.
+        """,
+    ]
+
     release: TypeAlias = Annotated[
         Union[str, Sequence[str]],
         "Release version identifier.",
     ]
+
     sample_set: TypeAlias = Annotated[
         str,
         "Sample set identifier.",
     ]
+
     sample_sets: TypeAlias = Annotated[
         Union[Sequence[str], str],
         """
         List of sample sets and/or releases. Can also be a single sample set or
         release.
         """,
     ]
+
     sample_query: TypeAlias = Annotated[
         str,
         """
-        A pandas query string to be evaluated against the sample metadata.
+        A pandas query string to be evaluated against the sample metadata, to
+        select samples to be included in the returned data.
         """,
     ]
+
+    sample_indices: TypeAlias = Annotated[
+        List[int],
+        """
+        Advanced usage parameter. A list of indices of samples to select,
+        corresponding to the order in which the samples are found within the
+        sample metadata. Either provide this parameter or sample_query, not
+        both.
+        """,
+    ]
```

Comment:
This parameter has been introduced to make typing easier when handling the case where samples are being selected via a list of indices rather than a pandas query.